front-end/src/app/shared/components/table-list-base/table-list-base.component.ts

```diff
@@ -80,16 +80,17 @@ export abstract class TableListBaseComponent<T> implements OnInit, AfterViewInit
     const first: number = event.first ? event.first : 0;
     const rows: number = event.rows ? event.rows : 10;
     const pageNumber: number = Math.floor(first / rows) + 1;
+    const params = this.getGetParams();
 
     // Determine query sort ordering
     let ordering: string = event.sortField ? event.sortField : '';
     if (ordering && event.sortOrder === -1) {
       ordering = `-${ordering}`;
     }
 
-    this.itemService.getTableData(pageNumber, ordering).subscribe((response: ListRestResponse) => {
+    this.itemService.getTableData(pageNumber, ordering, params).subscribe((response: ListRestResponse) => {
       this.items = [...response.results];
-      this.totalItems = response.count;
+      this.totalItems = this.items.length;
```

Comment:
Will this give the total number of records for the query and not just the number of records for the page?

Reply:
Reverted to the prior implementation, `this.totalItems = response.count`